### ISSUE TITLE
Can't Install

### ISSUE THREAD

**Issue by volomike** [jwintz/qchart.js#12]
I'm using Qt/C++ 5.5 (F/OSS LGPL version) on a Mac. I'm not able to get this installed into a QWidget type of application. I was hoping to use the QQuickWidget and load some QML in there, but in different attempts to get your code to work, I either end up with:

QChart.qml not found

or:

Chart is not a Type

What's the trick?

**Comment by volomike** [comment]
I figured out what my problem was. Originally, my QQuickWidget was loading a QML from resource. Well, when we do that, it appears to have a problem referencing other QML files stored in the same resource path, even though logically you would assume it would work.

To fix this, I added a QQuickWidget with an empty source property. Then, in my mainwindow.cpp of my widget application, I did this:

```
QString sQMLPath = QCoreApplication::applicationDirPath().append("/objects/qml/QChartGallery.qml");
ui->quickWidget->setSource(QString("file://").append(sQMLPath));
```

Then, in order to ensure that the Mac deployed this to the proper path when running the app, I had to edit my .pro file and add this in:

```
mac {
  Resources.files = objects
  Resources.path = Contents/MacOS
  QMAKE_BUNDLE_DATA += Resources
}
```

Then, I dropped all these project files into the objects/qml folder that I created in the project folder. When the application is run, it copies out objects (and subcontents) into the proper Contents\MacOS folder of the .app folder so that it can read the QML and show the chart.
